tests/cephadm/test_mon_stack_auth.py

```diff
@@ -0,0 +1,70 @@
+from api.cephapi.auth.auth import Auth
+from cli.cephadm.cephadm import CephAdm
+from cli.exceptions import OperationFailedError
+from utility.log import Log
+
+log = Log(__name__)
+
+HEADER = {
+    "Accept": "application/vnd.ceph.api.v1.0+json",
+    "Content-Type": "application/json",
+}
+
+
+def run(ceph_cluster, **kw):
+    """Verify redeploy for a specific service
+    Args:
+        **kw: Key/value pairs of configuration information to be used in the test.
+    """
+    config = kw.get("config", {})
+    installer = ceph_cluster.get_nodes(role="installer")[0]
+
+    # Get mgr services
+    out = CephAdm(installer).ceph.mgr.services()
+    username = config.get("username")
+    password = config.get("password")
+
+    for mon_stack in ["dashboard", "prometheus"]:
```

Comment:
Better to have separate methods to check if these services are enabled in utility

Reply:
added